src/components/panel-body/UEpage/router.jsx

```diff
@@ -100,48 +127,30 @@ const Router = () => {
             <GenericTile
               title={D.surveyUnitHousing}
               editable
-              icon={() => <MaterialIcons type="home" />}
-              editionIcon={() => (
-                <MaterialIcons
-                  type="pen"
-                  onClick={() => selectFormType(formEnum.ADDRESS, true)}
-                ></MaterialIcons>
-              )}
+              icon={housingIcon}
+              editionIcon={housingEditIcon}
```

Comment:
This will imply that `icon={<HousingIcon />}` ...

Reply:
You have 2 choice : 
* `icon={<HousingIcon />}` in GenericTile component (props `icon` only) and in the jsx {icon} 
* Icon={HousingIcon} in GenericTile props `Icon` and in the jsx <Icon />